tools/coqdep/lib/error.ml

```diff
@@ -33,13 +33,15 @@ let _ = CErrors.register_handler @@ function
     Some Pp.(str msg)
 
   | InvalidFindlibPluginName (f, s) ->
-    Some Pp.(str (Printf.sprintf "in file %s, %s is not a valid plugin name anymore." f s)
-      ++ str "Plugins should be loaded using their public name according to findlib," ++ spc ()
-      ++ str "for example package-name.foo and not foo_plugin." ++ spc ()
-      ++ str "If you are using a buid system that doesn't yet support the new loading method" ++ spc ()
-      ++ str "(such as Dune) you must specify both" ++ spc ()
-      ++ str "the legacy and the findlib plugin name as in:" ++ spc ()
-      ++ str "Declare ML Module \"foo_plugin:package-name.foo\".")
+    Some Pp.(str "in file " ++ quote (str f) ++ str "." ++ spc () ++ str "The name "
+      ++ quote (str s) ++ strbrk " is no longer a valid plugin name." ++ spc ()
+      ++ strbrk "Plugins should be loaded using their public name according to \
+      findlib, for example " ++ quote (str "package-name.foo") ++ str " and not "
+      ++ quote (str "foo_plugin") ++ str "." ++ spc () ++ strbrk "If you are \
+      using a buid system that does not yet support the new loading method \
+      (such as Dune) you must specify both the legacy and the findlib plugin \
+      name as in:" ++ spc ()
+      ++ str "      Declare ML Module \"foo_plugin:package-name.foo\".")
```

Comment:
Please include in the error link to docs, such as https://coq.github.io/doc/master/refman/proof-engine/vernacular-commands.html?highlight=declare#coq:cmd.Declare-ML-Module and probably https://github.com/ejgallego/coq-plugin-template and/or a `coq_makefile` example.

Reply:
While I'm not a real OCaml dev so I might be outside the target audience, I also think we should link _somewhere_ to an introduction to findlib/META/etc.; https://dev.realworldocaml.org/platform.html#building-an-executable-program introduces at least some of those concepts, but not enough to follow all the current discussions.

I propose:
1. the error should link to the manual;
2. the manual should link to tutorial and other findlib/etc references;
3. so that the manual can explain things more clearly and be improved — in this MR where easy, and separately later.